api/plugin.go

```diff
@@ -69,13 +68,59 @@ func (s *Server) SignPluginMessages(c echo.Context) error {
 		return fmt.Errorf("fail to unmarshal payroll policy, err: %w", err)
 	}
 
+	var payload v1.KeysignPayload
+	err = json.Unmarshal([]byte(req.Messages[0]), &payload)
```

Comment:
No , this is not correct , based on our discussion the other day , I was mean to change `KeysignRequest` to 
```
type KeysignRequest struct {
	PublicKey        string   `json:"public_key"`         // public key, used to identify the backup file
	Messages         v1.KeysignPayload `json:"messages"`           // Messages need to be signed
	SessionID        string   `json:"session"`            // Session ID , it should be an UUID
	HexEncryptionKey string   `json:"hex_encryption_key"` // Hex encryption key, used to encrypt the keysign messages
	DerivePath       string   `json:"derive_path"`        // Derive Path
	IsECDSA          bool     `json:"is_ecdsa"`           // indicate use ECDSA or EDDSA key to sign the messages
	VaultPassword    string   `json:"vault_password"`     // password used to decrypt the vault file
}
```